syft/frameworks/torch/torch_attributes.py

```diff
@@ -96,6 +96,7 @@ def __init__(self, torch: ModuleType, hook: ModuleType) -> None:
             "range",
             "save",
             "set_",
+            "set_num_threads",
```

Comment:
Is there an issue attached to this/what breaks?

Reply:
There is no particular issue attached to this but using "set_num_threads" gets hooked and breaks whenever called.